### PR TITLE
doc: Convert miscased Traffic Server references to |TS| macro

### PR DESCRIPTION
**Pull request by randall**

### CODE REVIEW

**Review by bneradt — DISMISSED** [review]
👍

**Review by bneradt — APPROVED** [review]

File: doc/developer-guide/api/functions/TSAcceptor.en.rst [hunk 1]
```diff
@@ -44,8 +44,7 @@ An acceptor thread listens for incoming connections and creates the virtual conn
 
 :func:`TSAcceptorGet` returns :type:`TSAcceptor` object that created :arg:`sslp`.
 
-:func:`TSAcceptorGetbyID` returns the :type:`TSAcceptor` object identified by :arg:`id`. :type:`TSAcceptor` represents the acceptor object created by the core
-traffic server.
+:func:`TSAcceptorGetbyID` returns the :type:`TSAcceptor` object identified by :arg:`id`. :type:`TSAcceptor` represents the acceptor object created by the server.
```
Comment (ywkaras) [inline doc/developer-guide/api/functions/TSAcceptor.en.rst:47]: Is this a mistake?   What server?

Comment (randall) [reply doc/developer-guide/api/functions/TSAcceptor.en.rst:47]: What is a core traffic server?

Comment (SolidWallOfCode) [reply doc/developer-guide/api/functions/TSAcceptor.en.rst:47]: Walt's right - that should be "proxy" or "core". "server" will cause people to think it is on a remote box.

Comment (bneradt) [reply doc/developer-guide/api/functions/TSAcceptor.en.rst:47]: We discussed this in the PR/issue meeting today, we suggest simply using `|TS|`.

Comment (ywkaras) [reply doc/developer-guide/api/functions/TSAcceptor.en.rst:47]: Can we just delete the word "core"?  "server" seems too ambiguous.

**Review by SolidWallOfCode — APPROVED** [review]

**Comment by zwoop** [comment]
Cherry-picked to v9.2.x

**Pull request merged**